client/core/wallet.go

```diff
@@ -300,6 +300,38 @@ func (w *xcWallet) LogFilePath() (string, error) {
 	return logFiler.LogFilePath(), nil
 }
 
+// AccelerateOrder uses the Child-Pays-For-Parent technique to accelerate an
+// order if the wallet is an Accelerator.
+func (w *xcWallet) AccelerateOrder(swapCoins, accelerationCoins []dex.Bytes, changeCoin dex.Bytes, requiredForRemainingSwaps, newFeeRate uint64) (asset.Coin, string, error) {
```

Comment:
Do the `xcWallet` methods need to be exported?

Reply:
Nope. Neither do some other ones.